data/generalhw_scripts/power_on_off_shelly.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script can operate shelly wifi switch plugs
+# See https://shelly-api-docs.shelly.cloud/gen1/#shelly-plug-plugs-relay-0
+
+shelly_ip=$1
+state=$2
+
```

I would check the number of args `$#` here to fail early if needed.